IBMigrationTool/main.swift

```diff
@@ -34,42 +34,96 @@ extension FileManager {
 /// Converts all `UIView+Displayable` runtime attributes to use the
 /// `CALayer` equivalents in the file at the given path
 /// - Parameter path: The file path to convert
-func migrateUserDefinedRuntimeAttributesInInterfaceBuilderFile(at path: String) {
+/// - Returns: Whether there were un-migrateable attributes in the file.
+func migrateUserDefinedRuntimeAttributesInInterfaceBuilderFile(at path: String) -> Bool {
     
-    print("Migrating user defined runtime attributes in \(path)")
+    print("=> Migrating user defined runtime attributes in \(path)")
     
     guard let migrator = InterfaceBuilderFileMigrator(filePath: path) else {
-        print("Failed to read data as string from: \(path)")
-        return
+        print("=> Failed to read data as string from: \(path)")
+        return false
     }
     migrator.migrate()
     
+    if !migrator.unmigratableMatches.isEmpty {
+        print("""
+            => Found umigrateable properties in \(path):
+
+            \(migrator.unmigratableMatches.compactMap({ (keyValue) -> String in
+                return "Line \(keyValue.key): \(keyValue.value)"
+            }).joined(separator: "\n"))
+            """
+        )
+    }
+    
     // Save file to disk!
     let newData = migrator.string.data(using: .utf8)
     do {
         let fileURL = URL(fileURLWithPath: path)
         try newData?.write(to: fileURL)
-        print("Wrote migrated file to \(path)")
+        print("=> Wrote migrated file to \(path)")
     } catch {
-        print("Failed to write migrated file to \(path)")
+        print("=> Failed to write migrated file to \(path)")
+    }
+    
+    return !migrator.unmigratableMatches.isEmpty
+}
+
+var pValue: String?
+let yesStrings = ["y", "Y", "yes", "YES"]
```

Comment:
Ideally only need "y" and "yes" then use `lowercased()` but happy to leave of course